### PR TITLE
Fix for nocontrol

### LINKED ISSUE #38: bam path from meta creates issues
### Have you checked the docs?

- [X] [nf-core website: troubleshooting](https://nf-co.re/usage/troubleshooting)
- [X] [nf-core modules documentation](https://nf-co.re/docs/contributing/modules)

### Description of the bug

GREP_SAMPLENAME takes input paths from path., this might be creating errors when one uses hidden paths. 

### Command used and terminal output

_No response_

### Relevant files

_No response_

### System information

_No response_

### PR DESCRIPTION
**Pull request by kubranarci**
- add cram/crai files to input list (assets/schema_input.json)
- add only a warn about the indices does not exists in converttovcf.json (bin/convertToStdVCF.py)
- change default arg for ENSEMBL VEP (conf/modules.config)
- upgrade samtools version for cram handling (modules/local/grep_samplename.nf)
- edit nextflow version to latest (nextflow.config)

### CODE REVIEW

File: conf/modules.config [hunk 1]
```diff
@@ -166,7 +166,7 @@ process {
         ]
     }
     withName: 'ENSEMBLVEP_VEP' {
-        //ext.args         ='--everything --filter_common --per_gene --total_length --offline'
+        ext.args         ='--everything --filter_common --per_gene --total_length --offline'
```
Comment (NagaComBio) [inline conf/modules.config:169]: Please exclude `--filter_common` and `--per_gene`, as we need all annotations for all variants, regardless of MAF. We also require all consensus data, not just the 'most severe' consensus for each variant.

**Comment by NagaComBio** [comment]
@kubranarci, Could you also add an update to the changelog?

**Review by NagaComBio — CHANGES_REQUESTED** [review]
Looks good. Just need an update regarding the VEP options and the changelog.

File: conf/modules.config (hunk 1 above)
Comment (kubranarci) [reply conf/modules.config:169]: these parameters are mainly default settings, update on those parameters has to be given by local config. But I will this as well for now.

**Review by NagaComBio — APPROVED** [review]

**Pull request merged**